README.md

```diff
@@ -2,3 +2,58 @@ Scalanus
 ========
 
 Yet another scripting language. Basically [Janus](https://github.com/mkaput/janus) done better, faster and funnier. Now in Scala. Much scalable. Very advanced. Such wow. Project for Scala Course at AGH UST.
+
+## TODO
+
+| | |
+|---|---|
+| 💔 | not implemented |
+| ⚙️ | in progress |
+| 🎉 | ready to go |
+
+### Language features
+
+| Feature | Grammar | IR  | Interpreter |
```

Comment:
w sumie o co chodzi z tym Grammar, IR i Interperter? bo jak kumam to grammar to chodzi o koncepcje a  interperter o zakodzenie ale co z tym IR?